airflow/providers/opsgenie/hooks/opsgenie.py

```diff
@@ -132,3 +132,33 @@ def close_alert(
         except OpenApiException as e:
             self.log.exception('Exception when closing alert in opsgenie with payload: %s', payload)
             raise e
+
+    def delete_alert(
+        self,
+        identifier: str,
+        identifier_type: Optional[str],
+        user: Optional[str],
+        source: Optional[str],
```

Comment:
```suggestion
        identifier_type: Optional[str] = None,
        user: Optional[str] = None,
        source: Optional[str] = None,
```
Assuming `None` is a valid value for these args and to be consistent with the codebase.

Reply:
https://github.com/apache/airflow/pull/23405/commits/8230b57dcfc2d23f39d4920c23a22efa9e58df9b